src/Command/CheckRequirements.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Command;
+
+use Manala\Config\Requirement\Requirement;
+use Manala\Config\Requirement\RequirementChecker;
+use Manala\Config\Requirement\RequirementViolationList;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Checks if the host's environment meets Manala's requirements (Vagrant, Ansible, etc.).
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class CheckRequirements extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('check_requirements')
```

Comment:
I'll suggest `requirements:check` instead (as the `security:check` command provided by the `sensiolabs/security-checker`)